src/v/cluster/partition.h

```diff
@@ -276,6 +276,16 @@ class partition {
         return _raft->abort_configuration_change(rev);
     }
 
+    uint32_t batch_max_bytes() const {
```

Comment:
Is this an already existing common pattern? i.e. exposing a topic property through partition abstraction

Reply:
>Is this an already existing common pattern? i.e. exposing a topic property through partition abstraction

yeh same question

Reply:
actually, you both are right, we do not need to propagate this all the way down to partition, i will refactor it. I was thinking that maybe we could skip one topic cfg lookup but we already do this hence there will be no difference.